From 63c10a0cad14cf26027892eae6fe6178f4178a87 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 17 Jul 2024 10:08:57 -0700
Subject: [PATCH 1/4] Fix plugin installation path

Paths already include the installation prefix; don't prepend it again.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 libobs/obs-nix.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libobs/obs-nix.c b/libobs/obs-nix.c
index 77c36be..fe8a048 100644
--- a/libobs/obs-nix.c
+++ b/libobs/obs-nix.c
@@ -51,7 +51,7 @@ const char *get_module_extension(void)
 
 static const char *module_bin[] = {
 	"../../obs-plugins/64bit",
-	OBS_INSTALL_PREFIX "/" OBS_PLUGIN_DESTINATION,
+	OBS_PLUGIN_DESTINATION,
 	FLATPAK_PLUGIN_PATH "/" OBS_PLUGIN_DESTINATION,
 };
 
-- 
2.45.1

